README.md

```diff
@@ -1,3 +1 @@
-# Northcoders News API
-
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+Two .env files must be created to use this repo a development and test .env with a PGdatabase and PGdatabase test respectively
```

Comment:
Be explicit about this - .env.test and .env.development and PGDATABASE - this is important as typos will prevent the connection to the db from working . Don't forget to advise them to npm i dotenv